client/state/selectors/get-current-user-marketing-price-group.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Internal dependencies
+ */
+import { getCurrentUser } from 'state/current-user/selectors';
+
+/**
+ * Returns the marketing price group of the current user.
+ *
+ * @param {object} state Global state tree
+ * @returns {string?} The price group slug
+ */
+export default ( state ) => {
+	const currentUser = getCurrentUser( state );
+
+	return ( currentUser && currentUser.meta.marketing_price_group ) || null;
```

Comment:
Small thing, but we could use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here:

> return currentUser?.meta.marketing_price_group
or the following if you want `null` returned instead of `undefined`
> return currentUser?.meta.marketing_price_group || null

Reply:
Thanks for the tip! I have updated the code in fa49661.